batch7crud-roll8/src/main/java/com/lftechnology/batch7crud/controller/CommonHttpServlet.java

```diff
@@ -18,4 +19,24 @@ protected int parameterValueAsInt(HttpServletRequest request, int index) {
     String[] paths = getPathParams(request);
     return Integer.parseInt(paths[index]);
   }
+
+  protected String getAction(HttpServletRequest request) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "getAction" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)